themes/bootstrap5/theme.config.php

```diff
@@ -157,10 +157,14 @@
              * for a strong inheritance safety net but this is not required.
              */
             'barcode' => 'FontAwesome:barcode',
+            'browzine-best' => 'FontAwesome:file-lines',
             'browzine-concern' => 'FontAwesome:exclamation',
+            'browzine-delivery' => 'FontAwesome:paper-plane',
             'browzine-issue' => 'Alias:format-serial',
+            'browzine-link-resolver' => 'FontAwesome:file',
             'browzine-pdf' => 'FontAwesome:file-pdf-o',
             'browzine-retraction' => 'FontAwesome:exclamation',
+            'browzine-view' => 'FontAwesome:file',
```

Comment:
I really have no good ideas on these new icons.  `file` works as a not-thinking-too-hard answer but I'm open to something better.  

Meanwhile `paper-plane` for article delivery is maybe too silly.  https://fontawesome.com/icons/paper-plane?f=classic&s=solid